test/reader.cpp

```diff
@@ -0,0 +1,62 @@
+
+#include "gtest/gtest.h"
+#include "../include/reader.h"
+#include "zim/archive.h"
+
+namespace kiwix
+{
+  /**
+   * This test file is written primarily to demonstrate how Reader is simply a
+   * wrapper over an archive. We will be dropping this wrapper soon.
+   **/
+  TEST (Reader, archiveWrapper) {
+    Reader reader("./test/zimfile.zim");
+    zim::Archive archive = *reader.getZimArchive();
+
+    std::ostringstream s;
+    s << archive.getUuid();
+
+    ASSERT_EQ(reader.getId(), s.str());
```

Comment:
This show us that libzim API is not so nice.
We should add a cast to string to `zim::Uuid` to avoid this.